lib/carma/models/metadata.rb

```diff
@@ -37,19 +37,23 @@ def veteran=(veteran_data_hash)
       end
 
       def primary_caregiver=(pc_metadata_hash)
-        @primary_caregiver = Caregiver.new(pc_metadata_hash)
+        @primary_caregiver  = if pc_metadata_hash.nil? # rubocop:disable Layout/SpaceAroundOperators, Layout/ExtraSpacing
+                                nil
+                              else
+                                Caregiver.new(pc_metadata_hash)
+                              end
       end
 
       def secondary_caregiver_one=(sc_one_metadata_hash)
-        @secondary_caregiver_one =  if sc_one_metadata_hash.nil?
+        @secondary_caregiver_one  = if sc_one_metadata_hash.nil? # rubocop:disable Layout/SpaceAroundOperators
```

Comment:
Why are the extra spaces/rubocop disable necessary?

Reply:
I was struggling to satisfy the rubocop rules for aligning `if` and `else` while keeping indentation at increments of 2 spaces. After another attempt, I was able to pass rubocop by using an odd number of spaces for indentation and keeping the conditional statement aligned. Does that work?